docs/toc.yml

```diff
@@ -580,8 +580,10 @@
       href: csharp/tutorials/exploration/interpolated-strings-local.md
     - name: Advanced scenarios for string Interpolation
       href: csharp/tutorials/string-interpolation.md
-    - name: Safely update interfaces with default interface members
+    - name: Safely update interfaces with default interface methods
```

Comment:
was that supposed to go in?

Reply:
Yes. The official feature name is "default interface methods", not "default interface members".  See [spec proposals](https://github.com/dotnet/csharplang/blob/master/proposals/csharp-8.0/default-interface-methods.md)

Reply:
Ah yeah, I think the referenced file contains "members" as well.
"members" was also used by engineers on GitHub a few times when ppl asked for what they mean with "DIM" so that stuck in my head. (like half a year ago)